packages/manager/modules/pci/src/projects/project/storages/containers/translations/Messages_fr_FR.json

```diff
@@ -27,8 +27,7 @@
 
   "pci_projects_project_storages_blocks_add_archive_label": "Créer un conteneur d'archive",
   "pci_projects_project_storages_blocks_add_object_label": "Créer un conteneur d'objets",
-  "pci_projects_project_storages_containers_offer_high_performance": "High Performance - S3",
-  "pci_projects_project_storages_containers_offer_standard_s3": "Standard - S3",
+  "pci_projects_project_storages_containers_offer_s3": "S3",
```

Comment:
**nitpick:** Can be maintained as a constant.

Reply:
The translation is already done on it. 
We could do this change in a future optimisation.